### PR TITLE
fix(contact-us):  resolved spacing and text problems

### PR DESCRIPTION
**Pull request by Mardecera**
# Summary

- [x] Resolved spacing and text problems.
- [x] Added language change in the information part.

### CODE REVIEW

File: src/components/ContactUs/ContactUs.tsx [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+import H2 from 'ui/Titles/H2'
+import useIntlMessages from 'hooks/useIntlMessages'
 
 const ContactUs: React.FC = () => {
+  const intl = useIntlMessages()
+
   return (
     <Section id="contact-us" size="lg" type="margin">
       <Container size="xl" isfullwidth>
-        <SectionTitle size="lg" text="Contactanos" />
```
Comment (CrhistianT7) [inline src/components/ContactUs/ContactUs.tsx:12]: delete styled component SectionTitle

File: src/components/ContactUs/ContactUsInformation/ContactUsContent/ContactUsContent.styles.ts [hunk 2]
```diff
@@ -8,21 +8,19 @@ import { FontWeight } from 'styles/typography'
...
-`
+  gap: 0.5rem;
 
-export const ContactUsQuestion = styled.p`
-  color: ${colors.primary.main};
-  font-weight: 500;
-  font-size: 1rem;
-  line-height: 1.5rem;
-`
+  & h4{
```
Comment (CrhistianT7) [inline src/components/ContactUs/ContactUsInformation/ContactUsContent/ContactUsContent.styles.ts:13]: wait for Gustavo titles refactor to change color propertly

File: src/components/ContactUs/ContactUs.tsx (hunk 1 above)
Comment (Mardecera) [reply src/components/ContactUs/ContactUs.tsx:12]: Other components outside of Contact Us are using it.

**Review by CrhistianT7 — APPROVED** [review]
Great Job!

**Pull request merged**